docs/core/setup/dependencies.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Core SDK and runtime dependencies for Windows, Linux, and macOS - .NET Core
+description: Details the operating system and CPU architecture prerequisites to install the .NET Core SDK and runtime on on Windows, Linux, and macOS.
+author: leecow
+ms.author: leecow
+ms.date: 11/06/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# .NET Core dependencies and requirements
+
+This article details which operating systems and CPU architecture are supported by .NET Core.
+
+## Supported operating systems
+
+::: zone pivot="os-windows"
+
+<!-- markdownlint-disable MD025 -->
+<!-- markdownlint-disable MD024 -->
+
+# [.NET Core 3.0](#tab/netcore30)
+
+The following Windows operating systems are supported with .NET Core 3.0:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                            | Version                        | Architectures   |
+| ----------------------------- | ------------------------------ | --------------- |
+| Windows Client                | 7 SP1+, 8.1                    | x64, x86        |
+| Windows 10 Client             | Version 1607+                  | x64, x86        |
+| Windows Server                | 2012 R2+                       | x64, x86        |
+| Nano Server                   | Version 1803+                  | x64, ARM32      |
+
+For more information about .NET Core 3.0 supported OS, distributions, and lifecycle policy, see [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md).
```

Comment:
OS is singular. OSes would be the plural which I don't like. This shows up more than one time in this PR.

```suggestion
For more information about .NET Core 3.0 supported operating systems, distributions, and lifecycle policy, see [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md).
```